src/integration-tests/bash/cleanup.sh

```diff
@@ -685,6 +689,8 @@ genericDelete "$g_arg1" "$g_arg2" "$g_arg3" -friendlyDelete
 genericDelete "$g_arg1" "$g_arg2" "$g_arg3" -forceDelete
 SUCCESS="$?"
 
+deleteNamespacesIT
+
```

Comment:
Unlike every other delete, this approach is done in foreground and has no retries.

Let me see if there's an easy alternative.

Reply:
Also, it is not specifying the same timeout/force parameters we use everywhere else.

Reply:
Finally, it is not affecting the return code for the script.

I'll see if I can come up with something more in-line with the spirit of the rest of the script.